PrivateCloud.DiagnosticInfo/PrivateCloud.DiagnosticInfo.psm1

```diff
@@ -2318,6 +2338,8 @@ function Get-SddcDiagnosticInfo
 
                     $LocalNodeDir = Get-NodePath $using:Path $using:NodeName
 
+                    $transcript = "$LocalNodeDir\transcript-SystemInfo.log"
+                    "$(Get-Date -f o) Gathering systeminfo.exe" > $transcript

```

Comment:
I'd recommend using Start-Transcript as you'll see in a few other places. Guaranteed 100% capture of anything that comes out. Since these are in a 1-per-node job I'm pretty sure it won't/can't interfere with the transcript being taken in the parent session.

Reply:
Yeah, in my initial version I used show-update in combination with start-transcript.  I rejected it because write-host would  end up getting captured by receive-job, (which is then emitted out onto the main output, which felt kind of noisy to me.   

So I ended up going with this approach, which is ugly.  Let me know if you have any thoughts on this.

Reply:
That's a good point about how much more logging this results in. I guess I'd still consider trying to bang this onto Show-Update so there's only one tight spot where the timestamp gluing gets done. I don't have an opinion on s v. o, but it should probably be consistent all the way through, which again is easy to make happen if all the output gluing is in one narrowly grouped set of Show-XYZ.

I'm still wondering if a transcript is just a good idea for everything else that might get spat out which would be useful to have collected in an unconflicted per-node log.

Reply:
e.g., Show-Update -Output to have it use Write-Output, which you can then redirect away, as opposed to Write-Host.

I did get attached to color highlighting, which is a lot of why this exists.

Reply:
I've updated it to use Show-Update and not receive-job, as we weren't storing it anywhere and so this avoids echoing back to the console

Reply:
That makes a lot of sense.